### PR TITLE
Direct Message Subject Header May Contain Multiple Subjects

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: src/NATS.Client/MsgHeader.cs [hunk 1]
```diff
@@ -249,6 +249,30 @@ public string[] GetValues(string name)
...
+        /// <summary>
+        /// Gets the last value for the specific key.
+        /// Will be null if the key is not found
+        /// </summary>
+        /// <param name="name"></param>
+        /// <returns></returns>
+        public string GetLast(string name)
+        {
+            string[] all = nvc.GetValues(name);
+            return all == null ? null: all[all.Length-1];
```
Comment (mtmk) [inline src/NATS.Client/MsgHeader.cs:273]: Length > 0 would be nice

Comment (mtmk) [reply src/NATS.Client/MsgHeader.cs:273]: I guess getvalues might return null in that case, so maybe not needed.

**Review by mtmk — CHANGES_REQUESTED** [review]
Looks good. Just wanted to check if a bounds check is necessary.

File: src/NATS.Client/MsgHeader.cs (hunk 1 above)
Comment (scottf) [reply src/NATS.Client/MsgHeader.cs:273]: I just did a test to verify. The backing NameValueCollection returns null if the key is not present so GetValues returns null as well.

**Review by mtmk — APPROVED** [review]
LGTM

**Pull request merged**